wolfcrypt/src/wc_lms_impl.c

```diff
@@ -0,0 +1,26 @@
+/* wc_lms_impl.c
```

Comment:
can we remove these stub files entirely, i.e. do it the way we do the asynccrypt files?

until this gets merged into mainline, it would be nice to be able to leave symlinks in place to the real versions without provoking `git`.

Reply:
async requires detection in configure and cmake.
We decided to have all the new algorithms done this way.
Let's leave it as is for now.

Reply:
yeah fair enough -- I had that thought too.